src/main/java/se/jiderhamn/classloader/leak/prevention/ClassLoaderLeakPreventor.java

```diff
@@ -2466,5 +2471,9 @@ boolean unregisterJettyJMXBean(ObjectName objectName) {
 		    return false;
 	    }
     }
-  }  
+
+    Class getClassFromWebApplicationParentClassLoader(String className) throws ClassNotFoundException {
+      return Class.forName(className, false, getWebApplicationClassLoader().getParent());
```

Comment:
Will always looking at the parent be backwards compatible, or would you want to fall back to Class.forName() if not found by parent?